qldbdriver/result.go

```diff
@@ -157,6 +174,11 @@ type IOUsage struct {
 	writeIOs *int64
 }
 
+// NewIOUsage creates a new instance of IOUsage.
```

Comment:
Why do we need this and NewTimingInformation. I don't think we need them.

Even if we do, they should not be exported.

Reply:
Mostly just for convenience. Since the `IOUsage` and `TimingInformation` structs take in pointers to ints, it's easier to use these constructors than to have to do this:

```go
readIOs := 0
writeIOs := 0
ioUsage := &IOUsage{&readIOs, &writeIOs}
```

Anyway I've unexported the constructors.

Reply:
Mostly just for convenience. Since these structs take in pointers to ints, its easier to use these constructors than to have to do this kinda thing every time:

```go
readIOs := 0
writeIOs := 0
ioUsage := &IOUsage{&readIOs, &writeIOs}
```

Reply:
The constructors have been un-exported.